rastervision/evaluation/classification_evaluator.py

```diff
@@ -37,4 +39,22 @@ def process(self, scenes, tmp_dir):
                 scene_evaluation = self.create_evaluation()
                 scene_evaluation.compute(ground_truth, predictions)
                 evaluation.merge(scene_evaluation)
+
+            if hasattr(label_source, 'source') and hasattr(
```

Comment:
Since this block of code only applies to semantic segmentation, you should move it into `SemanticSegmentationEvaluator`.

Reply:
Done.